Classes/Notifications/NotificationSectionController.swift

```diff
@@ -49,9 +49,11 @@ SwipeCollectionViewCellDelegate {
             cell.isRead = true
             client.markNotificationRead(id: object.id, isOpen: true)
         }
-
+        if NotificationSectionController.selectedCellId != object.id || UIDevice.current.userInterfaceIdiom == .phone{
```

Comment:
Why is there iPhone specific code in there?

Reply:
+1, let's remove

Reply:
the Details view reloading problem with IPad not in iPhone i am just bypassing the condition check to showDetailViewController and in iPad its checks for the id if not matching i am calling showDetailViewController in that way I don't need reload again.  or every time it click the cell it returning IssuesViewController new instance because of that it reloading again then probably we need to change the object creating style .